tutorials/plot_ica_from_raw.py

```diff
@@ -45,12 +44,13 @@
 # We pass a float value between 0 and 1 to select n_components based on the
 # percentage of variance explained by the PCA components.
 
-ica = ICA(n_components=0.95, method='fastica')
+ica = ICA(n_components=0.95, method='fastica', random_state=0, max_iter=100)
 
 picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=False,
                        stim=False, exclude='bads')
 
-ica.fit(raw, picks=picks, decim=3, reject=dict(mag=4e-12, grad=4000e-13))
+ica.fit(raw, picks=picks, decim=3, reject=dict(mag=4e-12, grad=4000e-13),
+        verbose='warning')  # low iterations -> does not fully converge
```

Comment:
FYI it doesn't converge with the default (200) or 1000 here, either, not sure if there is something better we can/should do

Reply:
yes fastica can easily fail to converge.

cc @pierreablin